System/RTETools.h

```diff
@@ -216,6 +216,17 @@ namespace RTE {
 	/// <param name="angleRadians">The angle in radians to be converted.</param>
 	/// <returns>The converted angle in degrees.</returns>
 	inline float RadiansToDegrees(float angleRadians) { return angleRadians / c_PI * 180.0F; }
+
+	/// <summary>
+	/// Rounds a float to a set fixed point precision (digits after decimal point) with option to always ceil or always floor the remainder.
+	/// </summary>
+	/// <param name="inputFloat">The input float to round.</param>
+	/// <param name="roundingMode">(0, 1, 2) What method of rounding to use for the result. 
```

Comment:
I don't think you really need to use <para> here, but by all means feel free to. I don't think it's used elsewhere, but maybe it should be lol. That said, you're missing it for roundingMode 2, and I'd suggest putting a `-` between the number and the text, like `<para>0 (default option) - System default.</para>`